docs/README.md

```diff
@@ -0,0 +1,14 @@
+# Scan-files Infrastructure
+
+![AWS infrastructure diagram.  Full text description follows.](architecture.png)
+
+1. Virtual Private Cloud (VPC) with three public and three private subnets.
+1. API Gateway (AGW) to allow inbound/outbound communication, protected by Web Application Firewall (WAF), as main user request entry point and has an Lambda API as its target.
```

Comment:
Does the AGW allow outbound communication?  I thought the only outbound was from the Step Function lambda to AssemblyLine.

Reply:
reworded. Thanks!